packages/backend/src/auth/auth.module.ts

```diff
@@ -8,8 +10,8 @@ import { UsersService } from './users/users.service';
 import { JwtStrategy } from './jwt.strategy';
 
 @Module({
-  imports: [TypeOrmModule.forFeature([UserEntity])],
+  imports: [TypeOrmModule.forFeature([UserEntity, DevicesEntity])],
```

Comment:
Czy dołączenie do modułu Auth DevicesEntity i NotificationService jest konieczne? Przez to moduł auth traci niezaleność